.ci/test.sh

```diff
@@ -173,3 +173,73 @@ matplotlib.use\(\"Agg\"\)\
     conda install -q -y -n $CONDA_ENV ipywidgets notebook
     jupyter nbconvert --ExecutePreprocessor.timeout=180 --to notebook --execute --inplace *.ipynb || exit -1  # run all notebooks
 fi
+
+if [[ $TASK == "r-pkg" ]]; then
+    #. set up R environment
+    R_LIB_PATH=~/Rlib
+    mkdir -p $R_LIB_PATH
+    cd $TRAVIS_BUILD_DIR/R-package
+    echo "R_LIBS=$R_LIB_PATH" > ${HOME}/.Renviron
+    echo 'options(repos = "https://cran.rstudio.com")' > ${HOME}/.Rprofile
+
+    export PATH="$R_LIB_PATH/R/bin:$PATH"
+    if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+        apt-get install \
+            gfortran-5 \
+            libcurl4-openssl-dev
+        update-alternatives \
+            --install /usr/bin/gfortran gfortran \
+            /usr/bin/gfortran-5 \
+            10
+
+        apt-get install \
+            texlive-latex-recommended \
+            texlive-fonts-recommended \
+            texlive-fonts-extra \
+            qpdf
+        if ! command -v R &> /dev/null; then
+            R_VER=3.6.1
+            cd $BUILD_DIR
+            wget https://cran.r-project.org/src/base/R-3/R-$R_VER.tar.gz
+            tar -xzf R-$R_VER.tar.gz
+            R-$R_VER/configure --enable-R-shlib --prefix=$R_LIB_PATH/R
+            make
+            make install
+        fi
+    fi
+
+    if [[ $TRAVIS_OS_NAME == "macos" ]]; then
+        brew install \
+            r \
+            qpdf
+        brew cask install \
+            basictex
+        export PATH="/Library/TeX/texbin:$PATH"
+        sudo tlmgr update --self
+        # sudo tlmgr install \
+        #     inconsolata \
+        #     helvetic
```

Comment:
will remove this commented code if it isn't necessary. Commented for now to test something